lib/ui/painting.dart

```diff
@@ -286,6 +287,26 @@ enum StrokeCap {
   square,
 }
 
+/// Styles to use for line joins.
+///
+/// Note that this only affects line joins for polygons drawn by
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-comments-dartdocs
In particular, avoid "Note that".